test/helpers/kubectl.go

```diff
@@ -150,11 +151,15 @@ func (kub *Kubectl) Logs(namespace string, pod string) *CmdRes {
 		fmt.Sprintf("%s -n %s logs %s", KubectlCmd, namespace, pod))
 }
 
-// ManifestsPath returns the the full path of manifests (DaemonSets, YAML files,
-// etc.) for the Kubernetes version specified in the environment. This path
-// is the path specified in the VMs, not the host of the Vagrant VMs.
-func (kub *Kubectl) ManifestsPath() string {
-	return fmt.Sprintf("%s/k8sT/manifests/%s", BasePath, GetCurrentK8SEnv())
+// ManifestsGet returns the full path of the given manifest for the specific
+// Kubernetes versions if exists, if not it returns the global manifest file.
+func (kub *Kubectl) ManifestsGet(manifestFilename string) string {
+	fullPath := fmt.Sprintf("%s/%s/%s", manifestsPath, GetCurrentK8SEnv(), manifestFilename)
+	_, err := os.Stat(fullPath)
+	if err == nil {
+		return fmt.Sprintf("%s/%s", BasePath, fullPath)
+	}
+	return fmt.Sprintf("%s/k8sT/manifests/%s", BasePath, manifestFilename)
```

Comment:
For consistency, do `os.Stat` on this path too. Maybe return an empty string if this file does not exist, or also return an error that is passed to the caller.

Reply:
Hi @ianvernon 

I do not agree here. I think that an error doesn't make sense: It's written by a dev and no user-facing.

On the other hand, if we return the full path we'll apply the full path, and we're going to see in the kub.Apply the path and the error, if we have an empty string, it'll be harder to debug.

Reply:
Can we just `panic()` instead with a clear error message?

Reply:
`kubectl.Apply` will raise an exception and it'll stop the progress of the test.

Reply:
@eloycoto OK, fine by me.